### ISSUE TITLE
Result Question

### ISSUE THREAD

**Issue by alvinsun724** [CRIPAC-DIG/LATTICE#1]
Hi, thanks for your amazing work.

Just want to ask the baby dataset result is showed as from the code :
{'precision': array([0.00545127, 0.00432502]), 'recall': array([0.05198949, 0.08228838]), 'ndcg': array([0.02913475, 0.03712578]), 'hit_ratio': array([0.05425559, 0.08593469]), 'auc': 0.0}

but it is quite different from what you reported in the paper, just wonder if this is the final code of your implementation?

**Comment by JinghaoZ** [comment]
> Hi, thanks for your amazing work.
> 
> Just want to ask the baby dataset result is showed as from the code : {'precision': array([0.00545127, 0.00432502]), 'recall': array([0.05198949, 0.08228838]), 'ndcg': array([0.02913475, 0.03712578]), 'hit_ratio': array([0.05425559, 0.08593469]), 'auc': 0.0}
> 
> but it is quite different from what you reported in the paper, just wonder if this is the final code of your implementation?

Hello, thanks for your attention to our work.

The first result is metric@10 and the second is metric@20. For example,  'recall': array([0.05198949, 0.08228838]) means recall@10=0.05198949 and recall@20=0.08228838.  In our paper, we reported recall@20=0.0829. Your result is very close to our reported one. The same goes for other metrics.

**Comment by alvinsun724** [comment]
> > Hi, thanks for your amazing work.
> > Just want to ask the baby dataset result is showed as from the code : {'precision': array([0.00545127, 0.00432502]), 'recall': array([0.05198949, 0.08228838]), 'ndcg': array([0.02913475, 0.03712578]), 'hit_ratio': array([0.05425559, 0.08593469]), 'auc': 0.0}
> > but it is quite different from what you reported in the paper, just wonder if this is the final code of your implementation?
> 
> Hello, thanks for your attention to our work.
> 
> The first result is metric@10 and the second is metric@20. For example, 'recall': array([0.05198949, 0.08228838]) means recall@10=0.05198949 and recall@20=0.08228838. In our paper, we reported recall@20=0.0829. Your result is very close to our reported one. The same goes for other metrics.

Thanks a lot for your kind reply. Just have one more question: 

the reg_loss set in the bpr_loss function in the main.py file is set as 0.0     Is it the contrastive auxiliary loss you presented in the 2.4 section (As I do not find the contrastive auxiliary loss you define in the code even the hyperparameter Beta as used in the 2.6 section in overall loss function) just a regularised loss and always set as 0.0 in your experiments settings like MICRO/feats, MICRO w/o.contrast and MICRO experiments?  

Thank you very much

**Comment by JinghaoZ** [comment]
> > > Hi, thanks for your amazing work.
> > > Just want to ask the baby dataset result is showed as from the code : {'precision': array([0.00545127, 0.00432502]), 'recall': array([0.05198949, 0.08228838]), 'ndcg': array([0.02913475, 0.03712578]), 'hit_ratio': array([0.05425559, 0.08593469]), 'auc': 0.0}
> > > but it is quite different from what you reported in the paper, just wonder if this is the final code of your implementation?
> > 
> > 
> > Hello, thanks for your attention to our work.
> > The first result is metric@10 and the second is metric@20. For example, 'recall': array([0.05198949, 0.08228838]) means recall@10=0.05198949 and recall@20=0.08228838. In our paper, we reported recall@20=0.0829. Your result is very close to our reported one. The same goes for other metrics.
> 
> Thanks a lot for your kind reply. Just have one more question:
> 
> the reg_loss set in the bpr_loss function in the main.py file is set as 0.0 Is it the contrastive auxiliary loss you presented in the 2.4 section (As I do not find the contrastive auxiliary loss you define in the code even the hyperparameter Beta as used in the 2.6 section in overall loss function) just a regularised loss and always set as 0.0 in your experiments settings like MICRO/feats, MICRO w/o.contrast and MICRO experiments?
> 
> Thank you very much

Hi, thanks for your attention to our work.
MICRO is our extended version of LATTICE which utilizes contrastive auxiliary task. The codes are available at https://github.com/CRIPAC-DIG/MICRO.
